docs/en/quick-answers/administering/share-assets-between-projects.md

```diff
@@ -9,20 +9,38 @@ lang: en
 
 It's not uncommon to have a project that's used as an Asset Library, containing assets that can be loaded into shots in other projects.
 
-To achieve this, you can add a tab to the [Loader app](https://support.shotgunsoftware.com/hc/en-us/articles/219033078) that lists assets from this Asset Library project. In order to do so, you'll have to define this in the Loader settings for the engine and environment you're working in. You'll potentially have to update this in multiple places.
+With the introduction of the Linked Projects Field on the Asset Entity, you can now add a single tab to the [Loader app](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Supervisor_Artist_sa_integrations_sa_integrations_user_guide_html#the-loader) that includes all Linked Projects. In order to do so, you'll have to define this in the Loader settings for the engine and environment you're working in. You'll potentially have to update this in multiple places. 
+
+```yaml
+- caption: Assets - Linked
+    entity_type: Asset
+    filters:
+      - [linked_projects, is, "{context.project}"]
+    hierarchy: [project.Project.name, sg_asset_type, code]
+```
+
+You can refer to the Alias and VRED Engine settings [in the tk-multi-loader2.yml configuration file](https://github.com/shotgunsoftware/tk-config-default2/blob/a5af14aefbafaec6cf0933db83343f600eb75870/env/includes/settings/tk-multi-loader2.yml#L343-L347) included with [tk-config-default2](https://github.com/shotgunsoftware/tk-config-default2) as it is the default behavior there.
+
+---
+
+The initial way to achieve cross-project sharing was to add a tab to the [Loader app](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Supervisor_Artist_sa_integrations_sa_integrations_user_guide_html#the-loader) that lists assets from a specific Asset Library project.
```

Comment:
Before introducing the Linked Projects field on Assets, the initial process for cross-Project sharing was to add a tab...